docs/PyPiPublishing.md

```diff
@@ -94,7 +94,7 @@ For detailed installation instructions, refer to the
 - Ubuntu 18.04 or later (64-bit)
 - Python 3.8 or later
 - Supported frameworks:
-  - PyTorch >=2.2, <2.4
+  - PyTorch >=2.2, <2.5
```

Comment:
@MaximProshin, FYI, to align with https://github.com/openvinotoolkit/nncf/blob/release_v2130/README.md?plain=1#L442